esptfa_arima/Test_Management/services/analysis_doc_service.py

```
@@ -43,6 +43,25 @@ def get_or_create_draft(idempotency_key: str, user: User, **kwargs):
 
 
 
+# ANALYSIS DOCUMENT
+def create_analysis_document(draft: TestDraft):
+    try:
+        # Create the analysis document
+        document = AnalysisDocument.objects.create(
```

Ruff (0.14.10)

Undefined name `AnalysisDocument`

(F821)